aspnetcore/signalr/javascript-client.md

```diff
@@ -93,6 +93,18 @@ Use the [configureLogging](/javascript/api/%40aspnet/signalr/hubconnectionbuilde
 
 [!code-javascript[Logging levels](javascript-client/sample/wwwroot/js/chat.js?range=9-12)]
 
+## Reconnecting Clients 
```

Comment:
As a rule thumb, avoid gerunds in titles and use sentence casing. 

Reconnecting Clients --> Reconnect clients

Reply:
i had to google [gerund](http://www.chompchomp.com/terms/gerund.htm), but that's a good suggestion!

Reply:
I had to do the same the first time I heard the word. :-)